src/main/java/core/basesyntax/HelloWorld.java

```diff
@@ -1,8 +1,19 @@
 package core.basesyntax;
 
-/**
- * Feel free to remove this class and create your own.
- */
+import core.basesyntax.util.FigureSupplier;
+
 public class HelloWorld {
+    public static void main(String[] args) {
+        FigureSupplier figureSupplier = new FigureSupplier();
+
+        for (int i = 1; i <= 6; i++) {
+            if (i < 4) {
+                figureSupplier.getRandomFigure((int)(Math.random() * 5) + 1).draw();
```

Comment:
`getRandomFigure` should not take any params, the random value should be created inside this method.

Reply:
![image](https://github.com/mate-academy/jv-oop-advanced/assets/68182494/038f8451-6ecf-4183-9393-1660ca01778d)
no params at all)